src/js/components/FormField/FormField.js

```diff
@@ -8,7 +8,12 @@ import React, {
 import styled, { ThemeContext } from 'styled-components';
 import { defaultProps } from '../../default-props';
 
-import { focusStyle, parseMetricToNum } from '../../utils';
+import {
+  containsFocus,
+  useForwardedRef,
+  focusStyle,
+  parseMetricToNum,
+} from '../../utils';
```

Comment:
Please use explicit imports and alphabetized order. e.g.
```suggestion
import {
  containsFocus,
} from '../../utils/DOM';
....
```